addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -330,6 +336,22 @@ var Wysiwyg = Widget.extend({
         if (reload) {
             window.location.reload();
         }
+    }
+    ,discardEditions: async function (reload = true) {
```

Comment:
Can you change 
```
}
,discardEditions...
```
to
```
},
discardEditions...
```


Why do you set `reload = true`?
We can reload all the time as we have no mechanism to reset the changes otherwise.